.github/ISSUE_TEMPLATE/questions-help-support.md

```diff
@@ -4,4 +4,12 @@ about: Do you need support?
 
 ---
 
+## Habitat-Sim version
+vx.x.x
+
+Habitat is under active development, and we advise users to restrict themselves to stable releases. Are you using the latest [release version](https://github.com/facebookresearch/habitat-sim/releases) of Habitat-Sim? If not, why not?
+(We may not be able to help with problems in earlier versions.)
```

Comment:
Maybe add something like "as it may already be fixed in later versions and earlier versions sometimes lack the more verbose logging needed for debugging" to explain that there is good reason for this.